scripts/hab_conf_transfer_to_Q.Rmd

```diff
@@ -0,0 +1,129 @@
+
+```{r extract_temp_cond_turb, echo=TRUE, include = TRUE, eval = FALSE}
+vars <- c('local_name', 'temperature_c', 'conductivity_m_s_cm', 'turbidity')
+
+fish_coll_data <- habitat_confirmations %>%
+  purrr::pluck("step_2_fish_coll_data") %>%
+  dplyr::select(dplyr::all_of(vars))
+```
+
+
+## Join extracted data to form_fiss_site_2023
+
+Add temp, cond, and turb data from `step_2_fish_coll_data` to `form_fiss_site_2023`.
+
+Here we use `all_of(c())` to deselect the target columns. We can't use vars because is includes the `local_name` which we don't want to remove. We could have made another vector of column names, but since its only used once I didn't think it was necessary, but please let me know if I should. `all_of(c())` will still throw errors if a column is missing.
+
+As explained here https://github.com/NewGraphEnvironment/fish_passage_skeena_2023_reporting/issues/64, the three downstream sites at Thompson Creek were sampled twice (two haul passes) and each haul pass has there own temp, cond, and turb data. Therefore in `form_fiss_prep1` there are 3 extra observations (68) compared to `form_fiss_site_raw` (65), because `123377_ds_ef1`, `123377_ds_ef2`, `123377_ds_ef3` are duplicated to represent each haul pass. There is no haul pass column in `form_fiss_site_2023` so when doing QA read the comments to identify which fiss site corresponds to each haul pass (we should think about adding a haul pass column to `form_fiss_site_2023` in the future).
+
+
+```{r add_temp_cond_turb, echo=TRUE, include = TRUE, eval = FALSE}
+
+form_fiss_prep1 <- dplyr::left_join(form_fiss_site_raw %>%
+                               #remove target columns that are in form_fiss_site_raw so we can add them from fish_coll_data
+                               dplyr::select(-dplyr::all_of(c('temperature_c', 'conductivity_m_s_cm', 'turbidity'))),
+                              fish_coll_data %>%
+                               dplyr::distinct(local_name, temperature_c, conductivity_m_s_cm, turbidity),
+                              by = "local_name") %>%
+  dplyr::relocate(temperature_c:turbidity, .after = "location")
+```
+
+## Burn to geopackage
+
+Now we will burn the updated `form_fiss_site_2023` to the QGIS project in `'/data_field/2023/form_pscis_2023.gpkg'`.
+
+```{r burn, echo=TRUE, include = TRUE, eval = FALSE}
+form_fiss_prep1 %>%
+  sf::st_write(paste0("~/Projects/gis/", dir_project, '/data_field/2023/form_fiss_site_2023.gpkg'), append=F, delete_dsn=T)
+```
+
+## Backup
+
+Let's backup `form_fiss_site_2023.gpkg` so we can commit the `Rdata` file to git. Then we will update
```

Comment:
Really the most important one we are committing is `form_fiss_site_2023.csv` .  

For now we are also versioning the `Rdata` file but that will likely end soon since the `Rdata` file always changes (due to metadata) where as the `csv` only changes when the underlying data changes.  We are keeping the Rdata file for now since the workflow is so knew and due to minor differences in syntax that happen when we convert to and from the gpkg to an R object (the sf object).  I think I used`waldo::compare` to understand those differences but I forget now. some of them relate to how a csv works (`comma separated value` - ie. it puts commas (or any other user specified separator) to organize the tabular data - rows and columns)